prime-router/src/main/kotlin/cli/tests/WatersAuthTests.kt

```diff
@@ -189,6 +191,7 @@ class OktaAuthTests : CoolTest() {
         val historyApiTest = HistoryApiTest()
         historyApiTest.outputToConsole = this.outputToConsole
         passed = passed and historyApiTest.runHistoryTestCases(testCases)
+        this.outputMsgs.addAll(historyApiTest.outputMsgs)
```

Comment:
Fixed an issue in several places where the tests weren't printing output properly when run without the --sequential option.